components/PomodoroTimer.tsx

```diff
@@ -169,17 +180,19 @@ export function PomodoroTimer() {
                     onComplete={() => onFinish()}
                     children={({ remainingTime }) => {
                         const minutes = Math.floor(remainingTime / 60)
+
                         const seconds = remainingTime % 60
+                        const seconds_text = seconds >= 10 ? seconds : `0${seconds}`
```

Comment:
Here you could use `.padStart` function from the string method, so you don't need the ternary.

```suggestion
                        const seconds = String(remainingTime % 60).padStart(2, '0');
```